Phoenix/Server/Source/main.cpp

```diff
@@ -14,12 +14,17 @@ int main(int argc, char* argv[])
     // Likely a single command from Quartz that launches the logger + any other critical tools
 
 	// ===== Load Voxel data / Load lua =====
-	voxels::BlockRegistry registry = voxels::BlockRegistry();
-	registry.registerBlock("core:dirt", "Dirt");
-	registry.registerBlock("core:cobble", "CobbleStone");
-	registry.registerBlock("core:stone", "Stone");
+	std::cout << "Program started \n";
+	//voxels::BlockRegistry *registry = registry->get();
+	std::cout << "register dirt \n";
+	voxels::BlockRegistry::get()->registerBlock("core:dirt", "Dirt");
+	voxels::BlockRegistry::get()->registerBlock("core:cobble", "CobbleStone");
+	voxels::BlockRegistry::get()->registerBlock("core:stone", "Stone");
 	// TODO: Replace these manual calls to register blocks with a call to run lua files
 
+	std::cout << voxels::BlockRegistry::get()->getDisplayName(1);
+	std::cout << std::to_string(voxels::BlockRegistry::get()->getBlockId("core::dirt"));
```

Comment:
This should probably use quartz logging functionality, instead of `cout`.

Reply:
Same as above, these are just proof that things are working until we can get proper unit tests created. We shouldn't be registering or accessing blocks directly in the main function.